client/my-sites/marketing/do-it-for-me/difm-lite-in-progress.tsx

```diff
@@ -35,6 +43,22 @@ function DIFMLiteInProgress( { siteId }: DIFMLiteInProgressProps ) {
 	const translate = useTranslate();
 	const dispatch = useDispatch();
 
+	const sitePurchases = useSelector( ( state ) => getSitePurchases( state, siteId ) );
+	const difmPurchase = sitePurchases.find(
+		( purchase ) => WPCOM_DIFM_LITE === purchase.productSlug
+	);
+
+	const moment = useLocalizedMoment();
+	let contentSubmissionDueDate: string | null = null;
+	if ( difmPurchase?.subscribedDate ) {
+		const subscribedDate = new Date( difmPurchase.subscribedDate );
+		subscribedDate.setDate( subscribedDate.getDate() + difmPurchase.refundPeriodInDays );
```

Comment:
At this point, `subscribedDate` actually becomes the due date for content submission, and it confused me a bit when I read this part of the code. 
One potential way is to make `contentSubmissionDueDate` here a Date object and later use `moment().format()` to serialize it when it's needed.